tests/common.py

```diff
@@ -1834,3 +1834,18 @@ def reset_translation_cache(hass: HomeAssistant, components: list[str]) -> None:
         for loaded_components in loaded_categories.values():
             for component_to_unload in components:
                 loaded_components.pop(component_to_unload, None)
+
+
+@lru_cache
+def get_quality_scale(integration: str) -> dict[str, str]:
```

Comment:
Can we improve the typing of the return value?
We know that the value of the dict can only be `done`, `todo` and  `exempt`

Reply:
@edenhaus this is updated